orchagent/qosorch.cpp

```diff
@@ -1253,6 +1253,20 @@ task_process_status QosOrch::handleQueueTable(Consumer& consumer)
                 SWSS_LOG_ERROR("Resolving scheduler reference failed");
                 return task_process_status::task_failed;
             }
+            else
```

Comment:
Probably I miss something here, why we didn't hit this case:

```
            if (ref_resolve_status::success == resolve_result)
            {
                ....
                else if (op == DEL_COMMAND)
                {
                    // NOTE: The map is un-bound from the port. But the map itself still exists.
                    result = applySchedulerToQueueSchedulerGroup(port, queue_ind, SAI_NULL_OBJECT_ID);
                }
               ...
```

what is the value of "resolve_result" during you test?